python/lsst/cp/verify/utils.py

```diff
@@ -0,0 +1,46 @@
+#
+# This file is part of cp_verify.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+__all__ = ['mergeStatDict']
+
+
+def mergeStatDict(statDictA, statDictB):
+    """Merge one dictionary of statistics with another.
+
+    Parameters
+    ----------
+    statDictA : `dict` [`str`, `dict`]
+        A dictionary of statistics.
+    statDictB : `dict` [`str`, `dict`]
+        Another dictionary of statistics.
+
+    Returns
+    -------
+    output : `dict` [`str`, `dict`]
+        A dictionary containing the union of the internal
+        dictionaries of ``statDictA`` and ``statDictB``, indexed
+        by the amplifier names.
+    """
+    for amp in statDictA.keys():
+        statDictA[amp].update(statDictB[amp])
```

Comment:
So this assumes that the list of `amp` keys is the same for both `statDict`s?  Does that warrant a comment (& possibly a check)?

Reply:
These dictionaries are created by the `CpVerifyStatsTask._emptyAmpDict(exposure)` method, which should remove the need to pass the detector along with the two dictionaries to ensure the amps match.

Reply:
Sure they are...there, but this function lives in a general `utils.py` file, so some specifics about assumptions/requirements of the input parameters seems warranted.

Reply:
Added raise for this case.